dialogs/measurement/depth_profile_numeric_limits.py

```diff
@@ -0,0 +1,73 @@
+# coding=utf-8
+"""
+Created on 21.3.2013
+Updated on 30.5.2018
```

Comment:
Maybe not copy-paste the entire blurb as is...

I think it is silly to even have these dates in the first place (that's what version control is for) but if you are going to use them, "Created on" should be the date the file was created and "Updated on" should be the last time it was modified.